<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPage</class>
 <widget class="QMainWindow" name="MyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5,2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QPushButton" name="pushButton_Profile">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
	margin-top:10px;
	margin-left:10px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
            </property>
            <property name="text">
             <string>PROFILE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>MY PAGE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item alignment="Qt::AlignRight|Qt::AlignBottom">
               <widget class="QPushButton" name="pushButton_addActivity">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                </property>
                <property name="text">
                 <string>ADD ACTIVITY</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item alignment="Qt::AlignRight|Qt::AlignTop">
               <widget class="QPushButton" name="pushButton_addPost">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                </property>
                <property name="text">
                 <string>ADD POST</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,4,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_title">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>TITLE : </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,3,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>FOLLOWERS LIST</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="styleSheet">
             <string notr="true">QListWidget {
    background-color: #333333;
    border: 1px solid #666666;
    color: #CCCCCC;
    font-family: Arial;
    font-size: 12px;
}

QListWidget::item {
    background-color: #444444;
    padding: 8px;
}

QListWidget::item:hover {
    background-color: #555555;
}

QListWidget::item:selected {
    background-color: #007ACC;
    color: #FFFFFF;
}

QListWidget::item:selected:!active {
    background-color: #007ACC;
    color: #FFFFFF;
}

QListWidget::item:selected:active {
    background-color: #005C99;
}
</string>
            </property>
            <property name="itemAlignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,3,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>POSTS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_Post">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:#333333;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1,1,1,1,1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QPushButton" name="pushButton_previousPost">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
              </property>
              <property name="text">
               <string>PREVIOUS POST</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item alignment="Qt::AlignRight|Qt::AlignVCenter">
             <widget class="QPushButton" name="pushButton_like">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
              </property>
              <property name="text">
               <string>LIKE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QLabel" name="label_likesCount">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>LIKES : </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item alignment="Qt::AlignRight|Qt::AlignVCenter">
             <widget class="QPushButton" name="pushButton_addComment">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
              </property>
              <property name="text">
               <string>COMMENT</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QLabel" name="label_commentsCount">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>COMMENTS : </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QPushButton" name="pushButton_memory">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
              </property>
              <property name="text">
               <string>MEMORY</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item alignment="Qt::AlignRight|Qt::AlignVCenter">
             <widget class="QPushButton" name="pushButton_nextPost">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
              </property>
              <property name="text">
               <string>NEXT POST</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
